src/microbe_stage/SpawnSystem.cs

```diff
@@ -444,11 +444,7 @@ private int DespawnEntities(Vector3 playerPosition)
     /// </summary>
     private void ProcessSpawnedEntity(ISpawned entity, Spawner spawnType)
     {
-        // I don't understand why the same
-        // value is used for spawning and
-        // despawning, but apparently it works
-        // just fine
-        entity.DespawnRadiusSquared = spawnType.SpawnRadiusSquared;
+        entity.DespawnRadiusSquared = spawnType.SpawnRadiusSquared + Constants.DESPAWN_RADIUS_OFFSET_SQUARED;
```

Comment:
This (and line 95) is technically incorrect mathwise. In math, (a + b)^2 = a^2 + 2ab + b^2, not a^2 + b^2. Of course this works too if you just figure out a fitting number for `Constants.DESPAWN_RADIUS_OFFSET_SQUARED`, but it may be harder to estimate a good value, and I think the math would make the offset effectively shrink if the spawn radius were increased. This could be solved by changing the formula or by adding the offset before squaring. From my testing it seems that the offset unsquared should be at least about 50 to prevent parts of bacterial colonies from spawning outside the despawn radius.

Reply:
I guess if  you want to consider the squared distance to be calculated from the final distance. I just added an extra offset on top of the calculated value to make despawning not happen immediately. Though perhaps I should increase the offset a bit?

Reply:
I upped the offset to that.